core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -560,25 +553,17 @@ class SubmissionsService(
           )
         )
       )
+      _ = SubmissionRequestValidation.staticValidation(submissionRequest, methodConfig)
 
       entityProvider <- getEntityProviderForMethodConfig(workspaceContext, methodConfig)
 
-      _ = validateSubmissionRootEntity(submissionRequest, methodConfig)
-
-      _ = submissionRequest.userComment.map(validateMaxStringLength(_, "userComment", UserCommentMaxLength))
-
       gatherInputsResult <- MethodConfigurationUtils.gatherMethodConfigInputs(ctx,
                                                                               methodRepoDAO,
                                                                               methodConfig,
                                                                               methodConfigResolver
       )
 
-      validationResult <- entityProvider.expressionValidator.validateExpressionsForSubmission(methodConfig,
```

Comment:
This no longer uses a future, so we can just use a simple assignment, and the exception is thrown right away.